### ISSUE TITLE
AlertingNG duplicate alerts

### ISSUE THREAD

**Issue by malejpavouk** [grafana/grafana#35474]
**What happened**: We have enabled new alerting and routed the alerts to webhook. We see the same notifications twice (both on fire and resolved). Although I must admit that sometimes the notification arrives only once (so the behavior seems to be unstable). We have an HA setup over Thanos and PostgreSQL.
Thanos series seem to be properly deduplicated (there is really just one time series)

**What you expected to happen**: Deduplication

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.0.0
- Data source type & version: Prometheus
- OS Grafana is installed on: Azure AKS, official Grafana 
- User OS & Browser: not relevant
- Grafana plugins: clean install
- Others:

**Comment by dsiperek-vendavo** [comment]
Alerts are NOT deduplicated if running replicas.  
![image](https://user-images.githubusercontent.com/70541548/122971859-5e675800-d344-11eb-904e-57a9c898aabc.png)

![image](https://user-images.githubusercontent.com/70541548/122971891-67f0c000-d344-11eb-82ee-0c65dce5ad19.png)

I did not include instance names labels for security reasons but they are the same.  This issue is one of a few issues we are facing in trying to adopt grafana 8 alerts.

**Comment by xens** [comment]
According to the documentation this is the expected behavior -> https://grafana.com/docs/grafana/latest/alerting/unified-alerting/#clustering so we'll need to wait for v8.1

**Comment by malejpavouk** [comment]
@xens Thanks for sharing the link. It seems that guys from Grafana updated the docs in this regard (or I must have missed that)

**Comment by xens** [comment]
Yeah that's what I've thought also :) looking forward the next release !

**Comment by dcharleston** [comment]
Is this the issue to follow for tracking progress on clustering and HA setups with unified alerting? I'm really looking forward to that feature and it doesn't look like its in the latest 8.1.3 release so I'm curious about the timeline. Thanks!

**Comment by oskar-christensson** [comment]
Been experiencing this issue on 8.5.9 from helm charts despite following documentation here:
https://github.com/grafana/grafana/issues/35474

Multiple (same amount as grafana instances) alerts arrive at webhook.
